paragon/build-tokens.js

```diff
@@ -0,0 +1,94 @@
+const StyleDictionary = require('style-dictionary');
+const path = require('path');
+const chroma = require('chroma-js');
+
+const { formattedVariables, fileHeader } = StyleDictionary.formatHelpers;
+
+const PGN_PREFIX = 'pgn';
+const BASE_BUILD_PATH = path.resolve(__dirname, 'build');
+
+const colorTransform = (token) => {
+  const { value, modify = [], original } = token;
+
+  if (original.value.endsWith('%')) {
+    return original.value;
+  }
+
+  let color = chroma(value);
+
+  if (modify?.length > 0) {
```

Comment:
[note] Add `?` to this conditional which allows us to clear out any existing `modify` configurations on the base design tokens if the consuming theme doesn't want that behavior (e.g., `"modify": null`)